### PR TITLE
OPS-179: Add hadolint checks

### PR DESCRIPTION
**Pull request by m-paradowsky**

### CODE REVIEW

**Review by gothrek22 — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -16,6 +16,17 @@ node('ynd') {
     GIT = utils.checkoutRepository('git@github.com:ynd-consult-ug/docker-ruby.git')
   }
 
+  try {
+    stage('Hadolint checks') {
+      utils.hadolintChecks("Dockerfile")
+    }
+  } catch(err) {
+    stage('Send slack notification') {
```
Comment (maksym-bordun-ynd) [inline Jenkinsfile:24]: Please add indents to the code inside the `catch` block as well.

File: Jenkinsfile [hunk 2]
```diff
@@ -16,6 +16,17 @@ node('ynd') {
...
+      slackSend channel: "#ops-notifications", message: "Build failed: ${env.JOB_NAME}. Hadolint checks failed. See logs for more information. <${env.BUILD_URL}>", color: "danger"
+    }
+    error "Hadolint checks failed. " + err
```
Comment (maksym-bordun-ynd) [inline Jenkinsfile:27]: GString can be used instead of concatenation, though not sure `err` output is actually needed here.

**Comment by maksym-bordun-ynd** [comment]
@gothrek22 perhaps multibranch pipelines should be used instead? So that hadolint checks will run for each PR.

File: Jenkinsfile (hunk 2 above)
Comment (m-paradowsky) [reply Jenkinsfile:27]: @gothrek22 said that it would be nice to have `err` output as well.

**Pull request closed without merging**